### ISSUE TITLE
Focus theme: Return key in the comment field while modifying cancels the action

### ISSUE THREAD

**Issue by cintek** [moinwiki/moin#1559]
When you modify a wiki page, the comment field is focused and you press the return key you cancel the modification. I would expect that you apply the changes when you press that key.

**Comment by wagner-intevation** [comment]
This is a bug of the new 'focus' theme (#1528 ), as it has a reverse order of input buttons:
1. cancel -> default
2. preview
3. submit
as opposed to the order in e.g. modernized:
1. submit -> default
2. preview
3. cancel

**Comment by UlrichB22** [comment]
Thanks for your support. 
Please close the issue when fixed by the latest commits.

**Comment by wagner-intevation** [comment]
The issue will be automatically closed once the PR is merged into master.

**Comment by UlrichB22** [comment]
I would like to close this issue. 
The mentioned fix is for repo [cintek:new-theme](https://github.com/cintek/moin/tree/new-theme).
@cintek, please try to merge.

**Comment by cintek** [comment]
The fix is now merged into the repo.
